PlayerCharacter.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using UnityEngine;
+
+class PlayerCharacter{
+  public string name;
+  public string className;
+  private string race;
+  private string subrace;
+  private string background;
+  
+  private int strength;
+  private int wisdom;
+  private int dexterity;
+  private int constitution;
+  private int intelligence;
+  private int charisma;
+  private int level;
+  private int maxhp;
+  public int currenthp;
+  public int armorClass;
+  private int initiative;
+  private int speed;
+  public currentSpeed;
+
+  private int strSaveProficiency;
+  private int dexSaveProficiency;
+  private int conSaveProficiency;
+  private int intSaveProficiency;
+  private int wisSaveProficiency;
+  private int chaSaveProficiency;
+
+  private int profiencyBonus;
+
+  private string[] armorProficencies;
+  private string[] weaponProficencies;
+
+  public string[] equipment;
+  public int[] spellSlots = new int[10];
+  private string[] spells;
+  
+  public string condition;
```

Comment:
Should also be an array.

Reply:
made into an array, also made immunities and resistances array